cypress/e2e/api/engageShere.cy.js

```diff
@@ -0,0 +1,144 @@
+describe('Listar clientes na aplicação EngageSphere', () => {
+  const CUSTOMERS_API_URL = `${Cypress.env('API_URL')}`
```

Comment:
É uma boa prática (a qual ajuda na legibilidade do código) deixar uma linha em branco entre a definição de variáveis e outras expressões.

```suggestion
  const CUSTOMERS_API_URL = `${Cypress.env('API_URL')}`
  
```

Reply:
A propósito, que tal definir o valor abaixo para a variável `CUSTOMERS_API_URL`?

`${Cypress.env('API_URL')}customers`

Dessa forma, a string `customers` não precisa ser repetida em todas as requisições.